### ISSUE TITLE
Support hoist-non-react-statics ^2.2.2

### ISSUE THREAD

**Issue by inoas** [eugene-manuilov/react-gettext#9]
After upgrading npm packages we got:
```
npm install
npm WARN react-gettext@0.3.2 requires a peer of hoist-non-react-statics@^1.2.0 but none was installed.
```

`"hoist-non-react-statics": "^2.2.2",` is recent

Could react-gettext support that recent version?

**Comment by eugene-manuilov** [comment]
Hi @inoas,

just released 0.3.3 version which has updated `hoist-non-react-statics` peer dependency. Now it should support both `^1.2.0` and `^2.0.0-0` versions. Here is how i declared it: `"hoist-non-react-statics": "^1.2.0 || ^2.0.0-0"`.

Let me know if it helps to you.

**Comment by inoas** [comment]
Thank you. Solved it on our side.
